core/src/main/java/io/grpc/internal/Internal.java

```diff
@@ -29,7 +29,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.grpc;
+package io.grpc.internal;
```

Comment:
Why move this to internal package? If we generate JavaDoc with only public APIs, then this won't show up.